lib/uart.toit

```diff
@@ -101,6 +101,9 @@ class Port implements reader.Reader:
     the data is written. The duration of the break signal is bit-duration * $break_length,
     where bit-duration is the duration it takes to write one bit at the current baud rate.
 
+  If not all bytes could be written without blocking, this will be indicated by
```

Comment:
Maybe add a comment explaining how to handle this?
Small code-sample.
Also mention the `Writer` class?